src/ert/run_models/everest_run_model.py

```diff
@@ -304,25 +304,27 @@ def _handle_errors(
         realization: str,
         fm_name: str,
         error_path: str,
+        fm_running_err: str,
     ) -> None:
         fm_id = f"b_{batch}_r_{realization}_s_{simulation}_{fm_name}"
         fm_logger = logging.getLogger("forward_models")
-        with open(error_path, encoding="utf-8") as errors:
-            error_str = errors.read()
-
+        if Path(error_path).is_file():
+            error_str = Path(error_path).read_text(encoding="utf-8") or fm_running_err
```

Comment:
When are we getting this from the file and when from the event?

Reply:
We are getting this from the file if a forward model step (`step_name`) fails while it is running. Then the error will be added to `ERROR` from  `step_name.stderr.#`. In this case the event error will contain something like `failed with exit code 1`, not very informative. And that is we are reading it from the file. 

If the scheduler is the one that cancels the step running either due to max_runtime being reached or due to the user cancelling the forward model execution. The `event` will have an informative error such as `The run is cancelled due to reaching MAX_RUNTIME` and there will not be any error in `step_name.stderr.#` or in `ERROR`